app/models/channel.rb

```diff
@@ -14,6 +14,7 @@ class Channel < ApplicationRecord
   SUBSCRIBER_COUNT = :subscriber_count
   ADVANCED_SORTABLE_COLUMNS = [YOUTUBE_VIEW_COUNT, TWITCH_VIEW_COUNT, VIDEO_COUNT, SUBSCRIBER_COUNT, FOLLOWER_COUNT].freeze
   BITFLYER_CONNECTION = "BitflyerConnection".freeze
+  GEMINI_CONNECTION = "GeminiConnection".freeze
```

Comment:
Nit: So I don't think we should store the polymorphic types in the models. I'd rather see the following to keep down code density.

```
if publisher.selected_wallet_provider_type == GeminiConnection.to_s
```

Thoughts? (This is a non-blocker)

Reply:
Done.